classes/email-templates/class-pmpro-email-template-admin-change-admin.php

```diff
@@ -0,0 +1,165 @@
+
+<p>!!membership_change!!</p>
+
+<p>Log in to your WordPress admin here: !!login_url!!</p>', 'paid-memberships-pro' );
+	}
+
+	/**
+	 * Get the email template variables for the email.
+	 *
+	 * @since TBD
+	 *
+	 * @return array The email template variables for the email (key => value pairs).
+	 */
+	public function get_email_template_variables() {
+		$user = $this->user;
+		// If the user no longer has a membership level, set the membership_change text to "Membership has been cancelled."
+		if ( ! pmpro_hasMembershipLevel( null,  $this->user->ID ) ) {
+			$membership_change = __( 'The user\'s membership has been cancelled.', 'paid-memberships-pro' );
+		} else {
+			$membership_change = __( 'You can view the user\'s current memberships from their Edit Member page.', 'paid-memberships-pro' );
+		}
+
+		$email_template_variables = array(
+			'membership_change' => $membership_change,
+			'subject' => $this->get_default_subject(),
+			'name' => $user->display_name,
+			'display_name' => $user->display_name,
+			'user_login' => $user->user_login, 
+			'user_email' => $user->user_email, 
+		);
+		return $email_template_variables;
+	}
+
+	/**
+	 * Get the email template variables for the email paired with a description of the variable.
+	 *
+	 * @since TBD
+	 *
+	 * @return array The email template variables for the email (key => value pairs).
+	 */
+	public static function get_email_template_variables_with_description() {
+		return array(
+			'!!membership_change!!' => __( 'Membership Level Change', 'paid-memberships-pro' ),
+			'!!subject!!' => __( 'The default subject for the email. This will be removed in a future version.', 'paid-memberships-pro' ),
+			'!!name!!' => __( 'The display name of the user whose membership was changed.', 'paid-memberships-pro' ),
+			'!!display_name!!' => __( 'The display name of the user whose membership was changed.', 'paid-memberships-pro' ),
+			'!!user_login!!' => __( 'The login name of the user whose membership was changed.', 'paid-memberships-pro' ),
+			'!!user_email!!' => __( 'The email address of the user whose membership was changed.', 'paid-memberships-pro' ),
+		);
+	}
+
+	/**
+	 * Get the email address to send the email to.
+	 *
+	 * @since TBD
+	 *
+	 * @return string The email address to send the email to.
+	 */
+	public function get_recipient_email() {
+		return get_bloginfo( 'admin_email' );
+	}
+
+	/**
+	 * Get the name of the email recipient.
+	 *
+	 * @since TBD
+	 *
+	 * @return string The name of the email recipient.
+	 */
+	public function get_recipient_name() {
+		//get user by email
+		$user = get_user_by( 'email', $this->get_recipient_email() );
+		return $user->display_name;
+	}
+}
+
+/**
+ * Register the email template.
+ *
+ * @since TBD
+ *
+ * @param array $email_templates The email templates (template slug => email template class name)
+ * @return array The modified email templates array.
+ */
+function pmpro_email_templates_admin_change_admin( $email_templates ) {
```

Comment:
Just a note here to document that we could use a closure (anonymous function) here and the code would be a bit tighter and we would avoid creating yet another prefixed function for the namespace...

However, using a closure here like this would make it harder to unhook an email template using remove_filter, which folks may want to do.

So well done. I agree with the use of the named function to add the template here, even if it cluttered the namespace a bit.